AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -899,6 +890,51 @@ - (void)transitionContext:(_ASTransitionContext *)context didComplete:(BOOL)didC
   _transitionContext = nil;
 }
 
+#pragma mark - Layout
+
+- (void)_completePendingLayoutTransition
+{
+  ASDN::MutexLocker l(_propertyLock);
+  if (_pendingLayoutTransition) {
+    [self _setCalculatedLayout:_pendingLayoutTransition.pendingLayout];
+    [self _completeTransition:_pendingLayoutTransition];
+    _pendingLayoutTransition = nil;
+  }
+  [self _completeLayoutCalculation];
+}
+
+- (void)_setCalculatedLayout:(ASLayout *)layout
```

Comment:
Think we should stay with `_setCalculatedLayout`. If you currently look into `ASDisplayNode` as well as the public API around returning a layout we always to `_layout` as `calculatedLayout`, e.g.:  `calculatedLayout`, `calculatedLayoutDidChange` or `constrainedSizeForCalculatedLayout`

I think it would actually makes sense to change the `_layout` instance variable to rename to `_calculatedLayout`. So we would be consistence across the code base ... @levi @appleguy any opinions about that?